app.js

```diff
@@ -0,0 +1,78 @@
+const options = ["rock", "paper", "scissors"];
+
+function getComputerChoice() {
+  const choice = options[Math.floor(Math.random() * options.length)];
+  return choice;
+}
+
+function checkWinner(playerSelection, computerSelection) {
```

Comment:
Using string everywhere can cause typo's so prefer to use them as minimum as possible, here you can create a const object for choices and result and then use it again and again.
Ex:

```
const CHOICES = {
  ROCK: "Rock",
  PAPER: "Paper",
  SCISSORS: "Scissors"
};

const RESULTS = {
  DRAW: "Draw",
  PLAYER: "Player",
  Computer: "Computer"
};
```

Reply:
Add use them as CHOICES.ROCK instead of "rock"